static/js/formHandler.js

```diff
@@ -115,6 +122,66 @@ const formSubmitHandler = () => {
         });
 };
 
+function voltageChartAjax(inputs, toGraph){
+  let $voltageChart = $("#voltage-Chart")
+  // myChart.destroy()
+  // console.log(toGraph)
+  $.ajax({
+          type: 'POST',
+          url: 'voltageChart',
+          dataType: 'json',
+          data: {
+            voltage: inputs[0].value,
+            length: inputs[1].value,
+            r_not: inputs[2].value,
+            r_a: inputs[3].value,
+            x: inputs[4].value,
+            force: inputs[5].value,
+            awg: inputs[6].value,
+            compute: 'force', //Change this later, currently just comparing vs force tho
+            toGraph: toGraph,
+            csrfmiddlewaretoken: $('input[name=csrfmiddlewaretoken]').val(),
+          },
+          success: function (data) {
+
+            var ctx = $voltageChart[0].getContext("2d");
+            if(window.line != undefined){
+              window.line.destroy()
+            }
+            window.line = new Chart(ctx, {
+             type: 'line',
+              data: {
+                labels: data.labels,
+                datasets: [{
+                  label: 'Force',
+                  backgroundColor: 'green',
+                  borderColor: 'green',
+                  data: data.data,
+                  fill: false,
+               }]          
+             },
+              options: {
+                tooltips: {
+                  mode: 'nearest',
+                  intersect: false,
+                },
+                responsive: true,
+                legend: {
+                  position: 'top',
+                },
+                title: {
+                  display: true,
+                  text: `Force vs ${toGraph}`
+                },
+                scales: {
+                   xAxes: [{ display: true, scaleLabel: { display: true, labelString: toGraph}}],
```

Comment:
Is this where @trevorliberty was talking about having r_not and/or r_a actually showing up with their subscripts?